pipeline-configs.yaml

```diff
@@ -7,16 +7,28 @@ jobs:
   #   jenkinsfile: Path and name of the Jenkinsfile, default Jenkinsfile
   #   after: [Automatically run this job after any of these jobs have been run]
 
+  OMERO-gradle-plugins-push:
+    description: Run scc merge and bump versions on omero-build
+    repository: ome/omero-build
+    branch: SPACEBRANCH
+    jenkinsfile: Jenkinsfile.merge
+    after:
+      - OMERO-gradle-plugins-build
```

Comment:
This shouldn't be here, `OMERO-gradle-plugins-build` contains the required `after: [OMERO-gradle-plugins-push]` to chain the jobs

Reply:
Doh. c-n-p error. Gone.